Sources/App/Core/Search.swift

```diff
@@ -230,14 +249,29 @@ enum Search {
         select = select
             .column(SQLFunction("LEVENSHTEIN", args: keyword, SQLBind(mergedTerms)),
                     as: levenshteinDist)
+        select = select
+            .column(ts_rank(vector: tsvector, query: tsquery), as: tsrankvalue)
         select = select
             .from(searchView)
         select = select
             .from(unnest(keywords), as: keyword)
         select = select
-            .where(keyword, ilike, SQLBind(searchPattern))
+            .from(plainto_tsquery(SQLBind(mergedTerms)), as: tsquery)
         select = select
-            .orderBy(levenshteinDist)
+            .from(TextSearch.toVector(keyword, weight: .b), as: tsvector)
+        select = select
+            .where(keyword, ilike, SQLBind(searchPattern))
+            // If we want to change the "does the search find it at all" mechanism to use
+            // the ts_vector (normalized english lexemes) representations, we can do so like:
+            //
+            //.where(to_tsquery(SQLBind(mergedTerms)), inVector, to_tsvector(keyword))
+            //
+            // Negative side effect is that terms prefixed with UI and NS (such as NSBezierCurve)
+            // won't be identified for the search term `bezier`, and non-english search terms
+            // (such as mandarin characters) completely fail to find anything.
+        select = select
+            //.orderBy(levenshteinDist)
+            .orderBy(tsrankvalue)
```

Comment:
I believe our ordering here is actually redundant, because we're sorting (client side) by the keyword frequency (descending).

We can remove both the levenshteinDist *and* the tsvector work here.

Reply:
Just another note: If you're searching for `ftp` you probably want that match on top:

<img width="200" alt="CleanShot 2022-08-07 at 17 23 32@2x" src="https://user-images.githubusercontent.com/65520/183298274-a68dadf7-9a73-4388-8a96-e9c6f82ebe66.png">

However, we can't `ORDER BY "tsrank", "frequency"`, because we don't have `frequency` available in that query. We do the sorting in Swift after fetching (what I called "client side" above, in the sense of "client of the db").

I think it's out of scope to try and address this and it's certainly not a regression introduced by this PR.